### DISCUSSION TITLE
how to remove duplicate values from query result

### DISCUSSION THREAD

**Discussion by coloz** [supabase/supabase#9312] [Questions]
example:
supabase.from('pre_forum_thread').select('*')
```
[
  {id:1,value:100},
  {id:1,value:100},
  {id:1,value:100},
  {id:2,value:100},
  {id:2,value:100},
  {id:3,value:12},
  {id:4,value:13}
]
```

after remove duplicate, i hope result:
[
  {id:1,value:100},
  {id:2,value:100},
  {id:3,value:12},
  {id:4,value:13}
]

**Comment by GaryAustin1** [comment]
No way from the client API.  Possible with a view or with an rpc call.
You could also just run a quick post process on the array.  Lots of hits on google.  https://stackoverflow.com/questions/36032179/remove-duplicates-in-an-object-array-javascript
